### PR TITLE
[16.0][MIG] sale_cancel_reason

### PR DESCRIPTION
**Pull request by hugo-cordoba**
16.0 mig sale_cancel_reason

### CODE REVIEW

File: sale_cancel_reason/tests/test_sale_cancel_reason.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        )
+        wizard.with_context(
+            active_model="sale.order", active_ids=self.sale_order.id
+        ).action_cancel()
+        self.assertEqual(
+            self.sale_order.state, "cancel", "the sale order should be canceled"
+        )
+        self.assertEqual(self.sale_order.cancel_reason_id.id, self.reason.id)
+
+    def setUp(self):
```
Comment (flalexg) [inline sale_cancel_reason/tests/test_sale_cancel_reason.py]: Move the setUp to setUpClass as it's way more performant since it runs only once.

**Review by flalexg — CHANGES_REQUESTED** [review]

**Review by flalexg — APPROVED** [review]

**Review by aliciagaarzo — APPROVED** [review]

**Comment by hugo-cordoba** [comment]
hi @rousseldenis, could you check this code and merge this development?

best regards

**Comment by hugo-cordoba** [comment]
Hi @pedrobaeza could you trigger the merge?

Regards.

**Comment by AaronHForgeFlow** [comment]
Next time, please check there are other PR open for the same module https://github.com/OCA/sale-workflow/pull/2401

**Comment by hugo-cordoba** [comment]
Hi @AaronHForgeFlow.

I don't see that he had commented that he was migrating the module in the migration task https://github.com/OCA/sale-workflow/issues/2215.

To avoid this on future occasions, it is advisable when you start migrating a module, to add a comment in the migration task so that the rest of us know about it.

Best regards!

**Comment by pedrobaeza** [comment]
Sorry, I didn't check either. I usually look for other PRs before doing this, but this time was direct.

**Comment by AaronHForgeFlow** [comment]
No worries, it happened to me in the past too. BR.

**Pull request merged**